lib/_emerge/actions.py

```diff
@@ -614,6 +614,28 @@ def action_build(
                         in trees[eroot]["root_config"].settings.features
                     )
                 ):
+                    if not trees[eroot]["root_config"].settings.get(
```

Comment:
Well, the one above was rather short but this really asks for a:

```
for x in ("BINPKG_GPG_SIGNING_GPG_HOME", "BINPKG_GPG_SIGNING_KEY"):
    ...
```

Reply:
OK, changed.